### ISSUE TITLE
txt module manual text box boundaries mixed axes

### ISSUE THREAD

**Issue by andr-ew** [andr-ew/nest_#19]
see this example, should expect different widths on x-axis being manually set but instead they are auto-sized

```
n = nest_ {
    _txt.label {
        y = 4,
        x = {
            { { 4, 24 }, { 28, 48 }, { 52, 72 }, { 90, 126 } },
            { { 4, 24 }, { 28, 48 }, { 52, 72 }, { 80, 126 } },
            { { 4, 24 }, { 28, 48 }, { 52, 72 }, { 80, 126 } }
        },  
        value = {
            { 'foo', 'bar', 'ding', 'bat' },
            { 'bar', 'foo', 'bat', 'ding' },
            { 'bat', 'ding', 'bar', 'foo' }
        },
        border = 15,
        padding = 3,
        margin = 3,
        flow = 'y'
        --cellsize = { 10, 10 }
    }   
} :connect { key = key, enc = enc, screen = screen }
```

the issue is in `txtpoint()`, the "fixed" mode is not independent per-axis

`local fixed = (d.x[2] ~= nil) and (d.y[2] ~= nil)`

**Comment by andr-ew** [comment]
this is probably fixed now,,, but i am too lazy to fix

**Comment by andr-ew** [comment]
fixed! 0d52eb0742396f5aed4a457b004178e27cbf8a2f
